ajax/libs/sticky-kit/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "sticky-kit",
+  "filename": "sticky-kit.min.js",
+  "version": "1.1.3",
+  "description": "A jQuery plugin for creating smart sticky elements",
+  "keywords": [
+    "sticky"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/leafo/sticky-kit.git"
+  },
+  "author": "leafo <leafot@gmail.com>",
+  "homepage": "http://leafo.net/sticky-kit/",
+  "npmName": "sticky-kit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "sticky-kit*"
```

Comment:
Because I downloaded it with pnpm.
And there is a ```Tupfile```, so I used ```"sticky-kit*"``` as the basepath.